execution.go

```diff
@@ -566,6 +559,10 @@ func newQueryExecution(client *GraphQLClient, schema *ast.Schema, tracer opentra
 func (e *QueryExecution) execute(ctx context.Context, plan *queryPlan, resData map[string]interface{}) []*gqlerror.Error {
 	e.wg.Add(len(plan.RootSteps))
 	for _, step := range plan.RootSteps {
+		if step.ServiceURL == "__bramble" {
```

Comment:
```suggestion
		if step.ServiceURL == internalServiceName {
```

worth extracting this to a `const` similar to the others in `schema.go`